demo/vl-map.html

```diff
@@ -15,7 +15,7 @@
 <body is="vl-body">
     <vl-demo-page data-vl-webcomponent="vl-map">
         <vl-demo data-vl-title="Kaart met verschillende GRB kaartlagen">
-            <vl-map id="map">
+            <vl-map id="map" data-vl-extent="[-73122.443418, -23915.714632, 347445.354629, 297624.297341]">
```

Comment:
waarom 2 demos met een eigen extent?
is 1 niet genoeg?

Reply:
Lijkt me inderdaad niet nodig bij deze demo.

Reply:
Zal overschot geweest zijn van een test